src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfLottery = 3;
```

Comment:
This number can be a class constant:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names